front/lib/api/chat.ts

```diff
@@ -9,6 +9,34 @@ import {
   front_sequelize,
 } from "../models";
 
+export async function getChatSessions(
```

Comment:
these functions in `front/lib/api` are meant to generally go from the Postgres DB (see front/lib/models.ts) to the typed objects (see front/lib/types). These types are useful in particular because they're shared backend and front-end